docker/docker-compose.ci.yml

```diff
@@ -8,10 +8,22 @@ services:
     command: ['-h=0.0.0.0', '-m=hello unlock save the web', '-i=1984']
   ganache-integration:
     restart: always
-    image: trufflesuite/ganache-cli:v6.4.3
+    build:
+      context: ./development
+      args:
+        - blocktime=1
+      dockerfile: ganache.dockerfile
+    environment:
+      CI: '${CI}'
+      ERC20_CONTRACT_ADDRESS: '0x591AD9066603f5499d12fF4bC207e2f577448c46'
+      LOCKSMITH_PURCHASER_ADDRESS: '0xe29ec42f0b620b1c9a716f79a02e9dc5a5f5f98a'
```

Comment:
Can we add the BLOCK TIME var based on '${BLOCKTIME} here please. This will bring flexibility and solve problems for the Nicks too.